lib/datasource/conda/index.ts

```diff
@@ -0,0 +1,65 @@
+import { logger } from '../../logger';
+import { ExternalHostError } from '../../types/errors/external-host-error';
+import { cache } from '../../util/cache/package/decorator';
+import { HttpError } from '../../util/http/types';
+import { joinUrlParts } from '../../util/url';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, Release, ReleaseResult } from '../types';
+import { datasource, defaultRegistryUrl } from './common';
+
+export class CondaDatasource extends Datasource {
+  static readonly id = datasource;
+
+  constructor() {
+    super(datasource);
+  }
+
+  override readonly customRegistrySupport = false;
+
+  override readonly defaultRegistryUrls = [defaultRegistryUrl];
+
+  override readonly caching = true;
+
+  @cache({
+    namespace: `datasource-${datasource}`,
+    key: ({ registryUrl, lookupName }: GetReleasesConfig) =>
+      `${registryUrl}:${lookupName}`,
+  })
+  async getReleases({
+    registryUrl,
+    lookupName,
+  }: GetReleasesConfig): Promise<ReleaseResult | null> {
+    logger.trace({ registryUrl, lookupName }, 'fetching conda package');
+
+    const url = joinUrlParts(registryUrl, lookupName);
+
+    const result: ReleaseResult = {
+      releases: [],
+    };
+    try {
+      const response = await this.http.get(url);
+
+      const packageJson = JSON.parse(response.body);
```

Comment:
Do not manually parse JSON, use `http.getJson` function and create a proper type.